tests/integration/test_snap.py

```diff
@@ -40,3 +40,15 @@ def test_snap_remove():
 
     snap.remove("charmcraft")
     assert get_command_path("charmcraft") == ""
+
+
+def test_snap_set():
+    lxd = snap.ensure("lxd")
+
+    assert lxd.get("ceph.external") == "false"
+    assert lxd.get("criu.enable") == "false"
```

Comment:
FYI, trying to `get` a key that was not set yet throws an error.

Reply:
That makes sense. I think that it should probably throw an Exception in the lib as well, though we can make it a nice custom one with a sensible error message.